.circleci/generate_config.sh

```diff
@@ -8,15 +8,21 @@ skfs=$?
 git diff --quiet HEAD $(git merge-base main HEAD) -- sql/ sqlparser/
 skdb=$?
 
-if (($skc == 0 && $skfs == 0 && $skdb == 0))
-then
-    circleci-agent step halt
-fi
+# if (($skc == 0 && $skfs == 0 && $skdb == 0))
+# then
+#     circleci-agent step halt
+# fi
```

Comment:
The reason for this was that CircleCI would choke on empty `workflows:` sections. Now that `check-format` is always present, it can safely be removed.